internal/pkg/service/stream/storage/level/local/diskwriter/network/config.go

```diff
@@ -47,6 +47,6 @@ func NewConfig() Config {
 		ShutdownTimeout:        30 * time.Second,
 		KCPInputBuffer:         8 * datasize.MB,
 		KCPResponseBuffer:      512 * datasize.KB,
-		PipelineBalancer:       RandomBalancerType,
+		PipelineBalancer:       RoundRobinBalancerType,
 	}
```

Comment:
Currently we have no reason to use random load balancer, ... both ignore isReady == false pipelines.